lib/view/my_crosswords.dart

```diff
@@ -191,10 +191,16 @@ class ModifyCrosswordNameScreenState extends State<ModifyCrosswordNameScreen> {
                     backgroundColor:
                         MaterialStateProperty.all<Color>(Colors.green),
```

Comment:
Better option is to set floating button such tick or something else. This button is unreadable. Also I think that you should remove "Anuluj" button. We have a navbar to back to the previous screen. A better solution is to scale the entire screen so that you don't have to scroll to press a button. Just scale the photo accordingly depending on the size of the phone. Although I still think that this view is unnecessary and it should be just a popup with the name of the crossword with the possibility of closing and a button to change the name.

Reply:
Do przedyskutowania